WalletWasabi/Blockchain/Transactions/TransactionHistoryBuilder.cs

```diff
@@ -37,30 +38,13 @@ public List<TransactionSummary> BuildHistorySummary()
 					continue;
 				}
 
-				DateTimeOffset dateTime;
-				if (foundTransaction.Height.Type == HeightType.Chain)
-				{
-					if (wallet.BitcoinStore.SmartHeaderChain.TryGetHeader((uint)foundTransaction.Height.Value, out SmartHeader header))
-					{
-						dateTime = header.BlockTime;
-					}
-					else
-					{
-						dateTime = DateTimeOffset.UtcNow;
-					}
-				}
-				else
-				{
-					dateTime = foundTransaction.FirstSeen;
-				}
-
+				var dateTime = foundTransaction.FirstSeen;
```

Comment:
We can just use the `FirstSeen` now, it seems to be properly set even for wallets created in 2019.01.

Reply:
ACK